modules/kmm-deploying-modules.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: CONCEPT
+[id="kmm-deploying-modules_{context}"]
+= Deploying kernel modules
```

Comment:
[Avoid using gerunds in concept titles](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-concepts)